crates/sui-bridge/src/orchestrator.rs

```diff
@@ -161,9 +177,11 @@ where
                 store
                     .insert_pending_actions(&actions)
                     .expect("Store operation should not fail");
-
-                // TODO: ask executor to execute, who calls `remove_pending_actions` after
-                // confirming the action is done.
+                for action in actions {
+                    submit_to_executor(&executor_tx, action)
```

Comment:
nit: I think keeping the comment either here or above `submit_to_executor` that it will call `remove_pending_actions` would be useful.